src/uu/sort/src/chunks.rs

```diff
@@ -7,6 +7,9 @@
 
 //! Utilities for reading files as chunks.
 
+#![allow(dead_code)]
+// Ignores non-used warning for `borrow_buffer` in `Chunk`
```

Comment:
This warning has been there for a while hasn't it? Would you mind splitting it out in a separate commit?